bookshelf-standard/2-structured-data/pom.xml

```diff
@@ -0,0 +1,141 @@
+  </properties>
+
+  <prerequisites>                       <!-- Optional, but suggested -->
+    <maven>3.3.9</maven>                <!-- Recommended minimum maven version -->
+  </prerequisites>
+
+  <!-- THINGS ONLY USED WHEN RUN LOCALLY -->
+  <profiles>
+    <profile>
+      <id>local</id>
+      <dependencies>
+        <dependency>
+          <groupId>com.google.cloud.sql</groupId>
+          <artifactId>mysql-socket-factory</artifactId>
+          <version>1.0.2</version>
+        </dependency>
+        <dependency>                        <!-- http://dev.mysql.com/doc/connector-j/en/ -->
+          <groupId>mysql</groupId>
+          <artifactId>mysql-connector-java</artifactId>
+          <version>5.1.40</version>  <!-- v5.x.x is Java 7, v6.x.x is Java 8 -->
+        </dependency>
+        <dependency>
+          <groupId>com.google.api-client</groupId>
+          <artifactId>google-api-client-appengine</artifactId>
+          <version>1.21.0</version>
+        </dependency>
+      </dependencies>
+    </profile>
+  </profiles>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>1.9.46</version>
+    </dependency>
+
+    <dependency>                        <!-- REQUIRED -->
+      <groupId>javax.servlet</groupId>  <!-- Java Servlet API -->
+      <artifactId>servlet-api</artifactId>
+      <version>2.5</version>
+      <scope>provided</scope>           <!-- Provided by the Jetty Servlet engine -->
+    </dependency>
+
+    <dependency>                        <!-- Java Server Pages -->
+      <groupId>javax.servlet</groupId>
+      <artifactId>jsp-api</artifactId>
+      <version>2.0</version>
+    </dependency>
+
+    <dependency>                        <!-- JSP standard tag library -->
+      <groupId>jstl</groupId>
+      <artifactId>jstl</artifactId>
+      <version>1.2</version>
+    </dependency>
+
+    <dependency>                        <!-- Apache Taglibs -->
+      <groupId>taglibs</groupId>
+      <artifactId>standard</artifactId>
+      <version>1.1.2</version>
+    </dependency>
+
+    <dependency>                        <!-- Google Core Libraries for Java -->
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava</artifactId>  <!-- https://github.com/google/guava/wiki -->
+        <version>20.0</version>
+    </dependency>
+
+    <dependency>                        <!-- http://www.joda.org/joda-time/ -->
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+      <version>2.9.4</version>
+    </dependency>
+
+  </dependencies>
+
+  <build>
+    <!-- Optional - for hot reload of the web application when using an IDE Eclipse / IDEA -->
+    <outputDirectory>${project.build.directory}/${project.build.finalName}/WEB-INF/classes</outputDirectory>
+    <plugins>
+
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
```

Comment:
Sigh - sorry about this, but they would like us to use the old cloud sdk version.

BTW - I don't see `appengine.maven.plugin` defined anywhere.